CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ All notable changes to this project are documented in this file.
 
  - In case of error, still attempt to run more pass to report more diagnostics
  - Fixed compiler panic when an unresolved alias has a binding
+ - Added `edited` callback to `SpinBox`
```

Comment:
It doesn't look like that this PR adds the edited callback. Is this intentional here?

Reply:
No, I think that's fine :). I was just curious. It would be really great to also have documentation for the new callback though (otherwise it doesn't exist for our users :-)